qualcoder/qualcoder.py

```diff
@@ -57,6 +57,7 @@
 from .view_av import DialogCodeAV
```

Comment:
I dont use relative imports. So I will revert these to absolute imports. I found t least on Windows that relative imports made the code unusable as modules could not be found.

Reply:
I can revert all to absolute imports.